src/main/scala/edu/ie3/simona/model/thermal/CylindricalThermalStorage.scala

```diff
@@ -159,37 +160,14 @@ case object CylindricalThermalStorage {
     *   energy
     */
   def volumeToEnergy(
-      volume: ComparableQuantity[Volume],
-      c: ComparableQuantity[SpecificHeatCapacity],
-      inletTemp: ComparableQuantity[Temperature],
-      returnTemp: ComparableQuantity[Temperature]
-  ): ComparableQuantity[Energy] =
-    volume
-      .multiply(c)
-      .multiply(returnTemp.subtract(inletTemp))
-      .asType(classOf[Energy])
+      volume: squants.Volume,
+      c: SpecificHeatCapacity,
+      inletTemp: squants.Temperature,
+      returnTemp: squants.Temperature
+  ): squants.Energy = {
+    val tempDiff = (returnTemp - inletTemp)
+    c.multiply(tempDiff, volume)
 
-  /** Equation from docs for the relation between stored heat and volume change.
-    *
-    * @param energy
-    *   available energy
-    * @param c
-    *   Specific heat capacity
-    * @param inletTemp
-    *   Inlet temperature
-    * @param returnTemp
-    *   Return temperature
-    * @return
-    *   volume
-    */
-  def energyToVolume(
-      energy: ComparableQuantity[Energy],
-      c: ComparableQuantity[SpecificHeatCapacity],
-      inletTemp: ComparableQuantity[Temperature],
-      returnTemp: ComparableQuantity[Temperature]
-  ): ComparableQuantity[Volume] =
-    energy
-      .divide(c.multiply(returnTemp.subtract(inletTemp)))
-      .asType(classOf[Volume])
```

Comment:
Thanks for asking! The method has not been used so far. `squants.Energy` does not provide any method to interact with the `SpecificHeatCapacity`- since they don't have it - in line with multiplying a temperature(delta). Option a) We could introduce some copy of this method like we did with `SpecificHeatCapacity`. Option b) remove this method :)  But surely there could be more options. If you have any suggestions I'm happy to include them.

Reply:
Reincluded this now.

Reply:
And removed again :) -> see review further down